tests/tests_fabric/plugins/collectives/test_torch_collective.py

```diff
@@ -236,8 +239,17 @@ def test_collectives_distributed(n):
     collective_launch(_test_distributed_collectives_fn, [torch.device("cpu")] * n)
 
 
-def _test_distributed_collectives_cuda_fn(strategy, collective):
-    collective.create_group()
+@skip_distributed_unavailable
+@pytest.mark.parametrize("n", (1, 2))
+@RunIf(skip_windows=True)
+@mock.patch.dict(os.environ, os.environ.copy(), clear=True)  # sets CUDA_MODULE_LOADING in torch==1.13
+def test_collectives_distributed_default_pg(n):
+    collective_launch(partial(_test_distributed_collectives_fn, create_group=False), [torch.device("cpu")] * n)
```

Comment:
What does this test that is not covered by the newly added `test_default_process_group`?

These tests are somewhat expensive and flaky. So we shouldn't add any unnecessary CI stress